tests/acceptance/10_files/14_immutable/00_immutable.cf

```diff
@@ -0,0 +1,64 @@
+body common control
```

Comment:
I would find it VERY helpful to have a comment near the top of each test saying what use case it is testing like:

- test that immutable file cannot be changed by a files promise without fsattrs attribute
- test that mutable file can be changed by a files promise without fsattrs attribute
- test that immutable file CAN be changed by a files promise which includes the fsattrs attribute

Reply:
Yup, totally agree 🚀